packages/config/src/handlers/formDraft/updateFormDraft/handler.ts

```diff
@@ -83,6 +83,9 @@ export async function modifyDraftStatusHandler(
       await clearHearthElasticInfluxData(request)
       if (newStatus === DraftStatus.DELETED) {
         await clearQuestionConfigs(event)
+        draft.history = []
+        draft.comment = ''
+        draft.version = 0
```

Comment:
So we actually don't have a draft with version 0. If we create a draft, it starts from version 1. Rather in the frontend if we don't have any drafts, as a fallback we show version 0. So for the `DELETED` case we need to just remove the draft for that particular event.

Reply:
Keeping as it is as per our offline discussion